data/migrations/20200426223901_music.js

```diff
@@ -0,0 +1,60 @@
+exports.up = async function (knex, Promise) {
+  await knex.schema.createTable("users", (tbl) => {
+    tbl.increments();
+    tbl.string("username", 255).unique().notNullable();
+    tbl.string("password", 255).notNullable();
+    tbl.string("email", 255).unique().notNullable();
+    tbl.string("first_name", 255).notNullable();
+    tbl.string("last_name", 255).notNullable();
+  });
+  await knex.schema.createTable("images", (tbl) => {
+    tbl.increments();
+    tbl.text("image_name", 255).notNullable();
+    tbl.text("image_category", 255).notNullable();
```

Comment:
```suggestion
    tbl.string("image_category", 255).notNullable();
```
Wondering why these are text not string?

Reply:
This has been updated for consistency - all applicable changed to string for type